qless/workers/__init__.py

```diff
@@ -46,10 +47,10 @@ def title(cls, message=None):
     @classmethod
     def divide(cls, jobs, count):
         '''Divide up the provided jobs into count evenly-sized groups'''
-        jobs = list(zip(*itertools.izip_longest(*[iter(jobs)] * count)))
+        jobs = list(zip(*_compat.izip_longest(*[iter(jobs)] * count)))
         # If we had no jobs to resume, then we get an empty list
         jobs = jobs or [()] * count
-        for index in xrange(count):
+        for index in range(count):
```

Comment:
I think it would be better to either use `six` or `python-future` to get a proper `range` in Python 2.